scripts/flattener.sh

```diff
@@ -1,4 +1,6 @@
-mkdir contracts/.flattened
-npx truffle-flattener contracts/DXswapRouter.sol > contracts/.flattened/DXswapRouter.sol
-npx truffle-flattener contracts/libraries/DXswapLibrary.sol > contracts/.flattened/DXswapLibrary.sol
-npx truffle-flattener contracts/libraries/DXswapOracleLibrary.sol > contracts/.flattened/DXswapOracleLibrary.sol
+mkdir -p contracts/.flattened
```

Comment:
Maybe a Hardhat script/task would be more "inclusive" here? I've not been using Windows in a while, but are `sh` files executable there?

Reply:
Your're right. But actually why do we want to flatten contracts? If just for verifying I will use `hardhat verify`. Or is there any other reason?

Reply:
I think it's really just for verifying. I think pure usage of `hardhat flatten` will be enough if we don't use the flattened output in any other part of the code.

Reply:
Removed with [83fbd6b](https://github.com/levelkdev/dxswap-periphery/pull/42/commits/83fbd6b689c2c5574d4663517d3e2e15278ddddb)